doc.js

```diff
@@ -33,10 +48,10 @@ define('doc', ['event'], function(event) {
 	var query = function(elements) {
 		var selectedElements = [];
 
-		if(elements){
-			if(elements instanceof Array) {
+		if (elements){
+			if (elements instanceof Array) {
```

Comment:
both have similiar funcionality, however instanceof returns the actual type whereas typeof returns a string. personally I rather use instanceof, hence it is easier to track human made mistakes, i.e. `if (data typeof 'undefnied') { ... }`.